src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -84,7 +86,11 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *       try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *       try // OK
+ *       (InputStream ignored = System.in) {}
```

Comment:
Single line please, we need to show regular good code, not a edgecase

Reply:
Changes Done